### PR TITLE
audioctl: Fix broken things; add unmute option

### PR DESCRIPTION
**Pull request by jyn514**

### CODE REVIEW

File: provision/overlay/bin/audioctl [hunk 1]
```diff
@@ -94,7 +92,6 @@ case "$VERB" in
 	mute)
 		case "$VOLUME_MANAGER" in
 			amixer)
-				amixer -q set Master 100%-
```
Comment (charlesdaniels) [inline provision/overlay/bin/audioctl:97]: This behavior was desired. Pulseaudio is weird - I've hit edge cases where simply muting an output device does not cause sound to stop playing.

File: provision/overlay/bin/audioctl [hunk 2]
```diff
@@ -82,7 +80,7 @@ case "$VERB" in
 	down)
 		case "$VOLUME_MANAGER" in
 			amixer)
-				amixer -q set Master 2dB-
+				amixer -q set Master 2-
```
Comment (charlesdaniels) [inline provision/overlay/bin/audioctl:83]: I had been meaning to rewrite these to use +- 2%. i am unclear on the behavior here - does `2-` imply a decrease by 2% of maximum volume?

File: provision/overlay/bin/audioctl [hunk 3]
```diff
@@ -104,6 +101,18 @@ case "$VERB" in
 				;;
 		esac
 		;;
+	unmute)
+		case "$VOLUME_MANAGER" in
+			amixer)
+				amixer -q set Master unmute
```
Comment (charlesdaniels) [inline provision/overlay/bin/audioctl:107]: This should also raise the volume again... maybe something like 30% or so?

**Review by charlesdaniels — CHANGES_REQUESTED** [review]

File: provision/overlay/bin/audioctl (hunk 2 above)
Comment (jyn514) [reply provision/overlay/bin/audioctl:83]: This decreases audio by 2dB. The previous command threw an error: 
```zsh
% amixer set Master 2dB- 
amixer: Invalid command!
```

File: provision/overlay/bin/audioctl (hunk 1 above)
Comment (jyn514) [reply provision/overlay/bin/audioctl:97]: Ah, this makes the `unmute` case more difficult.

File: provision/overlay/bin/audioctl (hunk 2 above)
Comment (charlesdaniels) [reply provision/overlay/bin/audioctl:83]: It worked as written on Ubuntu 16.04 with `amixer version 1.1.0`. However the `2-` variant appears to also work, and is thus more portable.

**Pull request closed without merging**